README.md

```diff
@@ -31,55 +10,18 @@ After you're finished please remove all the comments and instructions!
 
 </div>
 
-<!-- TABLE OF CONTENTS -->
+# 📖 [hello-rails-back-end] <a name="about-project"></a>
```

Comment:
- [ ] In this README, please add a link to your React Front-End. This is a project requirement.